src/components/ProposalsItemFooter.vue

```diff
@@ -29,16 +48,9 @@ const { getRelativeProposalPeriod, formatPercentNumber } = useIntl();
         )
       }}
     </span>
-    <template
-      v-if="
-        proposal.quorum &&
-        proposal.scores_total &&
-        !proposal.space.plugins?.quorum
```

Comment:
What happen to this `!proposal.space.plugins?.quorum` case ?

Reply:
I think it was previously added because we don't have a way to calculate quorum using plugin's quorum on proposals page, because we have to send lot of requests for each proposal, now that we store plugin's quorum on proposal level, we can show this even if a space use plugin quorum